modules/brainbrowser/css/volume-viewer-demo.css

```diff
@@ -46,6 +46,7 @@ canvas.slice-display {
   margin: 3px 5px;
 }
 
+/* Not currently used in Loris
```

Comment:
@jstirling91 This CSS file seems to be provided by brainbrowser team, so I treated it as a "library" file and was reluctant to remove that part. But if you say it's fine, I will certainly do so